monkey/infection_monkey/monkey.py

```diff
@@ -262,7 +262,7 @@ def _setup_agent_event_serializers(self) -> AgentEventSerializerRegistry:
         return agent_event_serializer_registry
 
     def _build_server_list(self, relay_port: int):
-        my_servers = map(str, self._opts.servers)
+        my_servers = list(map(str, self._opts.servers))
         relay_servers = [f"{ip}:{relay_port}" for ip in get_my_ip_addresses()]
         return my_servers + relay_servers
```

Comment:
We should return a set to not include duplicate IP's. It's a small fix, since we're already here

Reply:
This was the issue that caused the agent to fail. We concatenated map and list. 

I will try to use set.